angular-ngc/tools/test-setup.ts

```diff
@@ -0,0 +1,11 @@
+import { getTestBed } from '@angular/core/testing';
+import {
+  BrowserDynamicTestingModule,
+  platformBrowserDynamicTesting,
+} from '@angular/platform-browser-dynamic/testing';
+
+// First, initialize the Angular testing environment.
```

Comment:
Lets drop the "First, " since it does nothing else?

Reply:
Sure, done.